docs/docs.go

```diff
@@ -1,5 +1,4 @@
-// Package docs GENERATED BY SWAG; DO NOT EDIT
-// This file was generated by swaggo/swag
+// Package docs Code generated by swaggo/swag. DO NOT EDIT
```

Comment:
I think that the auto-generated file shouldn't be included in this case.

Reply:
I'm not sure why, but feel free to revert it if not needed.

Reply:
Apologies, I initially thought this change was unrelated to the PR and didn't need to be committed. However, I realized my Swaggo version was outdated. After installing the latest version, I confirmed this diff does occur.

I have no further issues at the moment. Thanks again for your contribution!